ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/StatusCountIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Icon } from '@patternfly/react-core';
+import { BarsIcon, CheckCircleIcon, SecurityIcon } from '@patternfly/react-icons';
+import pluralize from 'pluralize';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+
+type StatusCountIconProps = {
+    text: string;
+    status: 'pass' | 'fail' | 'other';
+    count: number;
+};
+
+function getStatusIcon(status, count) {
```

Comment:
You'll be able to take advantage of [this code](https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx#L114-L192) for those statuses, although it should be renamed since check and cluster detail pages will use it. There's potential to combine some of the logic but I think they are different enough to where it makes sense to keep them separated. `StatusCountIcon` is specifically for displaying a count of 'Pass', 'Fail', and 'Other' (combination of multiple). Where `getClusterResultsStatusObject` handles each status separately and displays a tooltip as well. I do think that should be converted into its own component as opposed to just being a util function though. You can see how it's being used in `CheckStatusModal.tsx`.